module/zfs/dmu_objset.c

```diff
@@ -193,8 +193,18 @@ compression_changed_cb(void *arg, uint64_t newval)
 	 */
 	ASSERT(newval != ZIO_COMPRESS_INHERIT);
 
-	os->os_compress = zio_compress_select(os->os_spa, newval,
-	    ZIO_COMPRESS_ON);
+	/* XXX: ALLAN: Do we need the mask here? */
```

Comment:
I think this should also be able to get answered/tested before merging...

Reply:
its no even my question. alan was asking himself

Reply:
I know, but his is basically a review comment and shouldn't be in the code.

Reply:
Just get rid of it, we don't review in code here.